test/integration/tests.js

```diff
@@ -372,13 +400,38 @@ test('Gets entries by creation order and id order', async (t) => {
     .map((item) => item.sys.contentType.sys.id)
     .filter((value, index, self) => self.indexOf(value) === index)
 
-  t.deepEqual(contentTypeOrder, ['1t9IbcfdCk6m04uISSsaIK', 'cat', 'contentTypeWithMetadataField', 'dog', 'human', 'kangaroo', 'testEntryReferences'], 'orders')
+  t.deepEqual(contentTypeOrder, ['1t9IbcfdCk6m04uISSsaIK', 'cat', 'catalog', 'contentTypeWithMetadataField', 'dog', 'human', 'kangaroo', 'testEntryReferences'], 'orders')
   t.ok(
     response.items[0].sys.id < response.items[1].sys.id,
     'id of entry with index 1 is higher than the one of index 0 since they share content type'
   )
 })
 
+test.only('Gets an entry that has resource links', async (t) => {
+  t.plan(3)
+  const response = await client.getEntry('xspaceEntry')
+
+  t.ok(response.sys, 'sys')
+  t.ok(response.fields, 'fields')
+  t.deepEqual(response.fields, {items: [
```

Comment:
is `items` here the name of the field?

Reply:
Yep see:

![Screenshot 2022-09-22 at 16 15 40](https://user-images.githubusercontent.com/11991830/191771445-f894571d-7262-4846-be05-5d7d162f5fb7.png)